### ISSUE TITLE
Platform Display not working with latest alpha

### ISSUE THREAD

**Issue by jes1417** [hyperion-project/hyperion.ng#918]
- [x] I confirm that this is an issue rather than a question.

## Bug report
running raspbian lite with retropie version 
```pi@retropie:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
Linux retropie 4.19.118-v7l+ #1311 SMP Mon Apr 27 14:26:42 BST 2020 armv7l GNU/Linux
```

Hyperion alpha 6 Platform grabbing is working 
![image](https://user-images.githubusercontent.com/38092747/88883654-8153bf80-d1fa-11ea-9969-1f0337638464.png)

when I install the new alpha 7 platform grabbing looks like this

![image](https://user-images.githubusercontent.com/38092747/88883777-be1fb680-d1fa-11ea-9069-f061187afd16.png)

I can switch back to alpha 6 and all is good with the platform grabber, notice the alpha 6 I am using is a fork I made with lordgrey's pr to add wled and nanoleaf improvements to test his new features.

#### Steps to reproduce
run alpha 6, platform grabber works as expected, run alpha 7, platform grabber is a garbled mess

#### What is expected?
Platform grabber to work like it did in alpha 6 

#### What is actually happening?
not sure, platform grabber is not working correctly

#### System

working
```
Hyperion Server: 
- Build:       master (GitHub-71bb257/48f453d-1594127917)
- Build time:  Jul  7 2020 14:34:43
- Git Remote:  https://github.com/jes1417/hyperion.ng
- Version:     2.0.0-alpha.6
- UI Lang:     auto (BrowserLang: en-US)
- UI Access:   advanced
- Avail Capt:  dispmanx,v4l2,framebuffer,qt

Hyperion Server OS: 
- Distribution: Raspbian GNU/Linux 10 (buster)
- Arch:         arm
- Kernel:       linux (4.19.118-v7l+ (WS: 32))
- Browser:      Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36 
```
not working

```
Hyperion Server: 
- Build:       (HEAD detached at ffa2fd45) (Paulchen-Panther-69e9fa6d/ffa2fd45-1595517126)
- Build time:  Jul 24 2020 15:55:19
- Git Remote:  https://github.com/hyperion-project/hyperion.ng
- Version:     2.0.0-alpha.7
- UI Lang:     auto (BrowserLang: en-US)
- UI Access:   expert
- Avail Capt:  dispmanx,v4l2,framebuffer,qt

Hyperion Server OS: 
- Distribution: Raspbian GNU/Linux 10 (buster)
- Arch:         arm
- Kernel:       linux (4.19.118-v7l+ (WS: 32))
- Browser:      Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36 
```

**Comment by tihoangyeudau** [comment]
I confirm this issue

**Comment by Lord-Grey** [comment]
Closed, as duplicate to issue #913 and fixed by PR #906

**Comment by jes1417** [comment]
Thank you, I have Compiled a new build with #906 and can confirm the platform grabber works with the pi4 and retropie however I also tried to install this version on a pi3 with only raspbian lite and the UI wont load so I am not sure whats not working, I reinstalled the alpha release on the pi3 and the ui comes up as expected.
